lib/addons/p5.dom.js

```diff
@@ -1877,6 +1877,66 @@
    *
    * @method stop
    * @return {Object|p5.Element}
+   * @example
+   * <div><code>
+   *
+   * //This example both starts 
+   * //and stops a sound sample
+   * //when the user clicks the canvas
+   *
+   * //We will store the p5.MediaElement
+   * //object in here
+   * var ele;
+   *
+   * //while our audio is playing,
+   * //this will be set to true
+   * var sampleIsPlaying = false;
+   *
+   * function setup() {
+   *   //Here we create a p5.MediaElement object
+   *   //using the createAudio() function.
+   *   ele = createAudio('assets/beat.mp3');
+   *   setMessageAndBackground("Click to play!", 200);
+   * }
+   *
+   * function mouseClicked() {
+   *   //here we test if the mouse is over the
+   *   //canvas element when it's clicked
+   *   if(mouseX >= 0 && mouseX <= width &&
+   *     mouseY >= 0 && mouseY <= height) {
+   *     
+   *     if(sampleIsPlaying) {
+   *       //if the sample is currently playing
+   *       //calling the stop() function on
+   *       //out p5.MediaElement will stop
+   *       //it and reset its current
+   *       //time to 0 (i.e. it will start
+   *       //at the beginning the next time
+   *       //you play it)
+   *       ele.stop();
+   *
+   *       sampleIsPlaying = false;
+   *       setMessageAndBackground("Click to play!", 200);
+   *     } else {
+   *       //loop our sound element until we
+   *       //call ele.stop() on it.
+   *       ele.loop(); 
+   *
+   *       sampleIsPlaying = true;
+   *       setMessageAndBackground("Click to stop!", 200);
+   *     }
+   *   }
+   * }
+   *
+   * //simply set the message and background shade of grey
+   * //on the canvas. Included to make the example
+   * //above more readable.
+   * function setMessageAndBackground(message, backgroundColor) {
+   *   background(backgroundColor);
+   *   textAlign(CENTER);
```

Comment:
couldn't these two lines get set in setup() then the text() line could replace the setMessageAndBackground() call in the mousClicked() function

Reply:
Hi @lmccart 

I had the setMessageAndBackground() function in order to try simplify the repeated setting of the background and text. To be honest, I was worried about it (which is why it had that comment preamble about readability).

What I didn't consider was just not changing the background from light to dark gray. I've done that now and have pulled out the setMessageAndBackground() function.
I think it's better as an example now.

Thanks so much for the feedback!